src/sentry/api/endpoints/project_transaction_names.py

```diff
@@ -53,8 +54,12 @@ def get(self, request: Request, project) -> Response:
                     "unique_transaction_names": transaction_names
                     if return_all_names
                     else len(transaction_names),
-                    "rules_redis": get_redis_rules(project),
-                    "rules_projectoption": get_rules(project),
+                    "rules_redis": rule_store.get_redis_rules(
+                        ClustererRuleNamespace.TRANSACTIONS, project
```

Comment:
nit: This might be a good time to move the namespace and project args to the `__init__` method of the stores. I don't remember why I did not do that in the first place.

Reply:
I'm not sure if I understand. Isn't this already the case?

Reply:
Never mind, I was confused because I thought `rule_store` was an instance of `RedisRuleStore`.

Reply:
Yes, I was confused by the same thing. I think we should improve this API at some point, but that's out of scope for now.